cnd/Cargo.toml

```diff
@@ -37,6 +42,7 @@ paste = "0.1"
 pem = "0.7"
 rand = "0.7"
 reqwest = { version = "0.10", default-features = false, features = ["json", "native-tls"] }
+rustc-hex = "2.1.0"
```

Comment:
We already depend on the `hex` crate, would be good to use that one instead of bringing in a 2nd dependency.